health.go

```diff
@@ -80,6 +81,7 @@ func NewDefaultHealthCheck() *HealthCheck {
 		GracePeriodSeconds:     30,
 		IntervalSeconds:        10,
 		TimeoutSeconds:         5,
+		DelaySeconds:           15,
```

Comment:
Sorry, I did not mean to say that we should not support the parameter (we definitely should). I'm just not sure if we should extend `NewDefaultHealthCheck()`. It looks like you could define a custom value for `DelaySeconds` by creating a health check (either an empty one or based off of `NewDefaultHealthCheck`) and set the value accordingly?